src/globals/grid/__tests__/grid-test.js

```diff
@@ -8,12 +8,38 @@
  */
 
 const { types } = require('node-sass');
-const { convert, renderSass } = require('../../../../tools/jest/scss');
+const { convert, createSassRenderer } = require('@carbon/test-utils/scss');
+
+const sassRender = createSassRenderer(__dirname);
+const render = content =>
+  sassRender(`
+$css--font-face: false;
+$css--helpers: false;
+$css--body: false;
+$css--use-layer: false;
+$css--reset: false;
+$css--typography: false;
+$css--plex: false;
+${content}
+`);
+const renderClassic = content =>
+  render(`
+$feature-flags: (components-x: false, breaking-changes-x: false, grid: false);
+${content}
+`);
+
+const isClassic = async () => {
+  const { calls } = await renderClassic(`
+@import '../../scss/functions';
+$t: test(feature-flag-enabled('braking-changes-x'));
```

Comment:
```suggestion
$t: test(feature-flag-enabled('breaking-changes-x'));
```

is this a typo? appears a few times